teacher.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Teacher < Person
+  def initialize(specialization)
+    super()
+    @specialization = specialization
+  end
```

Comment:
- [ ] The Teacher class inherits from the Person class. Kindly include the other parameters necessary to create a person.

  ```
  def initialize(specialization, age, name = 'Unknown', parent_permission: true)
    super(age, name, parent_permission: parent_permission)
    @specialization= specialization
  end
  ```